Documentation/planning/multi-sdk-band-support.md

```diff
@@ -0,0 +1,258 @@
+- There are components of the 1xx band that are used in the runtime. There are "sdk-like" components present in the runtime (analyzers, generators, etc.) that are dependent upon "sdk" functionality. Substituting in a 2xx aligned roslyn, for instance, would result in a product that differs from Microsoft's build.
+
+### What artifacts should be passed with --with-artifacts/--with-packages?
+
+Today, `--with-packages` is used to pass the previously source-built packages. This serves as a feed containing packages that the .NET SDK internally depends on, as well as bootstrap input packages for the current build iteration (e.g. arcade). The source-build infrastructure will read this set of packages and produce a package version props file for each repo, depending on the dependencies listed in the `Version.Details.xml` file. If multiple packages exist with the same ID in this package source, the *latest* is chosen. 
+
+When performing a source-build of a product, there are 4 required input set of input artifacts:
+
+1. The .NET SDK to use
+2. The NuGet artifacts supporting that SDK
+3. The non-SDK NuGet artifacts required to bootstrap any component that depends on assets built *later* in the build. (e.g. arcade depends on itself to build, but is not provided by the .NET SDK).
+4. The input artifacts for components **not** built in the the product source build. Examples include:
+   - Prebuilts
+   - In a subsetted VMR case, components built in *another* VMR build.
+
+***Note: When dealing with just a 1xx SDK, full build, with no prebuilts, #4 is an empty set, and #2 and #3 are the previously-source-built artifacts passed with --with-packages.***
+
+When building a VMR, a distro maintainer will combine the sets of artifacts from #2, #3 and #4 into a single directory and pass it with --with-artifacts. Below, distro maintainer workflow scenarios are presented. The following abbreviations are used:
+
+- PSB - Previously source-built (from a previous servicing/preview iteration, or from a bootstrap build)
+- CSB - Current source-built (just built, e.g. from a 1xx VMR build).
+
+### Previews and major release GA builds
+
+**No Change**
+
+A distro maintainer sees no change here. Until a few months **after** GA, only the 1xx band ships. The workflow remains the same as today. Clone the VMR branch/tag in question, build using previously source-built artifacts and SDKs.
+
+### A distro maintainer wishing to only support the 1xx SDK
+
+**No change**
+
+### A distro maintainer wishing to build the initial release (n00) of a 2xx+ band SDK
+
+The initial release of a 2xx+ band SDK should be straightforward. It is unlikely (though possible) that the initial 2xx band would depend on a 2xx SDK to build. If this is not the case, then a distro maintainer performs the following actions, and no bootstrap will be required:
```

Comment:
Will this be tested/covered somewhere in our CI? If so, would we have the script that does this that we could provide?

Reply:
Yes, like we do with the 1xx band today, we would verify that the 2xx band can be built. What the required inputs are may differ based on repo requirements. I'll add a section on verification.